packages/scss/src/components/_grid.scss

```diff
@@ -131,7 +131,7 @@
 
 /* SM - Columns */
 
-@media only screen and (min-width: $sm) {
+@media only screen and (min-width: _get($luTheme, "breakpoints.sm")) {
 	.grid-sm {
 		flex-basis: 0;
```

Comment:
As I saw many flex usages on some BU I thought it was ok. 
So... what's our IE compatibility requirements?